src/shell/lean.cpp

```diff
@@ -434,6 +436,8 @@ int main(int argc, char ** argv) {
     std::string native_output;
     optional<std::string> c_output;
     optional<std::string> root_dir;
+    buffer<string_ref> args;
```

Comment:
We should probably call it something like `forwarded_args` to avoid confusion

Reply:
Agreed.